requirements.txt

```diff
@@ -3,3 +3,4 @@ molecule
 mock
 pytest-rpc
 moleculerize
+pytest-ordering
```

Comment:
Should this have a version constraint added to the constraints.txt file?

Reply:
@johnduarte updated